Fallback in AbstractFxmlView.getBundleName() is never called

Currently (in master and 1.4.1) `de.felixroske.jfxsupport.AbstractFxmlView#getBundleName`
returns the value if the bundle attribute regardless of whether it is set or not.
So users have to explicitly declare a bundle if they use internationalized strings in their `.fxml` markup.
e.g.: `@FXMLView(bundle="demo.projects.ui.projects")`, etherwise an exception will be thrown:
```
...
Caused by: javafx.fxml.LoadException: No resources specified.
/home/tom/dev/repos/gh/thomasdarimont/spring-labs/springboot-javafx-support-jpa-demo/target/classes/demo/projects/ui/projects.fxml:15

	at javafx.fxml.FXMLLoader.constructLoadException(FXMLLoader.java:2597) ~[jfxrt.jar:na]
	at javafx.fxml.FXMLLoader.access$100(FXMLLoader.java:103) ~[jfxrt.jar:na]
	at javafx.fxml.FXMLLoader$Element.resolvePrefixedValue(FXMLLoader.java:421) ~[jfxrt.jar:na]
	at javafx.fxml.FXMLLoader$Element.processValue(FXMLLoader.java:363) ~[jfxrt.jar:na]
	at javafx.fxml.FXMLLoader$Element.processPropertyAttribute(FXMLLoader.java:325) ~[jfxrt.jar:na]
	at javafx.fxml.FXMLLoader$Element.processInstancePropertyAttributes(FXMLLoader.java:235) ~[jfxrt.jar:na]
	at javafx.fxml.FXMLLoader$ValueElement.processEndElement(FXMLLoader.java:767) ~[jfxrt.jar:na]
	at javafx.fxml.FXMLLoader.processEndElement(FXMLLoader.java:2823) ~[jfxrt.jar:na]
	at javafx.fxml.FXMLLoader.loadImpl(FXMLLoader.java:2532) ~[jfxrt.jar:na]
	at javafx.fxml.FXMLLoader.loadImpl(FXMLLoader.java:2441) ~[jfxrt.jar:na]
	at javafx.fxml.FXMLLoader.load(FXMLLoader.java:2409) ~[jfxrt.jar:na]
	at de.felixroske.jfxsupport.AbstractFxmlView.loadSynchronously(AbstractFxmlView.java:159) ~[springboot-javafx-support-1.4.1.jar:na]
	... 12 common frames omitted
```

It seems that [this check](https://github.com/roskenet/springboot-javafx-support/blob/55885ce4124f00dce0fc4bb0da4750c16b879930/src/main/java/de/felixroske/jfxsupport/AbstractFxmlView.java#L357) `if (!StringUtils.isEmpty(annotation)) {` 
in `de.felixroske.jfxsupport.AbstractFxmlView#getBundleName` is wrong.

It should rather be `if (!StringUtils.isEmpty(annotation.bunde())) {`

--- a/src/main/java/de/felixroske/jfxsupport/AbstractFxmlView.java
+++ b/src/main/java/de/felixroske/jfxsupport/AbstractFxmlView.java
@@ -354,15 +354,15 @@ public abstract class AbstractFxmlView implements ApplicationContextAware {
 	 * @return the bundle name
 	 */
 	private String getBundleName() {
-		if (!StringUtils.isEmpty(annotation)) {
-			final String lbundle = annotation.bundle();
-			LOGGER.debug("Annotated bundle: {}", lbundle);
-			return lbundle;
-		} else {
+		if (StringUtils.isEmpty(annotation.bundle())) {
 			final String lbundle = getClass().getPackage().getName() + "." + getConventionalName();
 			LOGGER.debug("Bundle: {} based on conventional name.", lbundle);
 			return lbundle;
 		}
+
+		final String lbundle = annotation.bundle();
+		LOGGER.debug("Annotated bundle: {}", lbundle);
+		return lbundle;
 	}
 
 	/**

--- a/src/test/java/jfxtest/annotated/AnnotatedUnconventionalNameTest.java
+++ b/src/test/java/jfxtest/annotated/AnnotatedUnconventionalNameTest.java
@@ -15,7 +15,7 @@ import de.roskenet.jfxsupport.test.GuiTest;
 import javafx.scene.control.Button;
 @RunWith(SpringRunner.class)
 @SpringBootTest
-public class AnnotatedUnconventionalName extends GuiTest {
+public class AnnotatedUnconventionalNameTest extends GuiTest {
 	
 	@Autowired
 	private Annotated buttonsView;
@@ -25,12 +25,13 @@ public class AnnotatedUnconventionalName extends GuiTest {
         init(buttonsView);
     }
 	
-	@Test
-	/* 
+	/*
 	 * Asserts that we can find a view if the class name doesn't end with 'view'
+	 * see https://github.com/roskenet/springboot-javafx-support/issues/38
 	 */
+	@Test
 	public void showsI18nText() throws Exception {
 	   Button theButton = (Button) find("#theButton");
-	   assertThat(theButton.getText(), is("The Button Text"));
+	   assertThat(theButton.getText(), is("The default Button Text"));
 	}
 }

--- a/src/test/resources/jfxtest/annotated/annotated.fxml
+++ b/src/test/resources/jfxtest/annotated/annotated.fxml
@@ -5,6 +5,6 @@
 
 <Pane maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="400.0" prefWidth="600.0" xmlns="http://javafx.com/javafx/8.0.112" xmlns:fx="http://javafx.com/fxml/1">
    <children>
-      <Button fx:id="theButton" layoutX="262.0" layoutY="187.0" mnemonicParsing="false" text="The Button Text" />
+      <Button fx:id="theButton" layoutX="262.0" layoutY="187.0" mnemonicParsing="false" text="%thebuttontext" />
    </children>
 </Pane>

--- a/src/test/resources/jfxtest/annotated/annotated.properties
+++ b/src/test/resources/jfxtest/annotated/annotated.properties
@@ -1,0 +1,1 @@
+thebuttontext=The default Button Text

